docs/README-EDITING.md

```diff
@@ -0,0 +1,174 @@
+| `metaTitle`    | The page's SEO meta title.                                 | No       |
+| `tags`         | Tags used by the documentation search engine.              | No       |
+
+Frontmatter example:
+
+```
+---
+title: Introduction
+permalink: /next/api/
+canonicalUrl: /api/
+---
+```
+
+### Sitemap frontmatter
+
+The [`vuepress-plugin-sitemap`](https://www.npmjs.com/package/vuepress-plugin-sitemap) lets us use additional tags to customize the documentation site map:
+
+```
+---
+sitemap:
+  exclude: false
+  changefreq: hourly
+---
+```
+
+## Editing the documentation
+
+When editing the documentation content, follow the guidelines below.
+
+### Editing the `next` documentation version
+
+Draft files to be included in the documentation's next version are kept in the `next` directory.
+
+The `next` version of the documentation is available only locally and on the staging server.
+
+To display the `next` version in a browser:
+1. Start a [local Handsontable documentation server](./README.md#getting-started-with-handsontable-docs).
+2. In your browser, go to http://localhost:8080/docs/next/.
+
+### Editing a published documentation version
+
+To edit an already-published documentation version, go to the required [`<semver.version>` directory](./README.md#handsontable-docs-directory-structure) (e.g. `9.0`).
+
+To display a published documentation version in a browser:
+1. Start a [local Handsontable documentation server](./README.md#getting-started-with-handsontable-docs).
+2. In your browser, go to http://localhost:8080/docs/{semver.version}/.
+
+If you're editing the `latest` version (a version with the largest `<semver.version>` number), remember to make the same edits to the `next` version as well.
+
+## Editing the API reference
+
+The API reference is generated automatically from the source code, into the `/next/api/` directory.
+
+To edit the API reference, go into the source code and change the JSDoc comments.
+
+To generate the API reference output, run the following command from the `/docs/` directory:
+
+```bash
+npm run docs:api
+```
+
+### JSDoc links
+
+As we want to support case-insensitive URLs, follow these guidelines when editing the JSDoc links:
+
+* Use lower case characters.
+* To separate words before `#`, use `-`.
+
+```js
+// Wrong:
+/** {@link Options#autoColumnSize}. */
+
+// OK: 
+/** {@link options#autocolumnsize Options#autoColumnSize}. */
+
+
+// Wrong:
+/** {@link HiddenRows} */
+
+// OK:
+/** {@link hidden-rows HiddenRows} */
+```
+
+## Documentation versioning
+
+To create a new version of the Handsontable documentation:
+
+* From the `handsontable/docs` directory, run:
+```bash
+npm run docs:version <semver.version>
+# for example:
+# npm run docs:version 9.0
+```
+
+To remove an existing version of the Handsontable documentation:
+
+* Remove the required version's [directory](./README.md#handsontable-docs-directory-structure):
+```bash
+rm -rf ./<semver.version>
```

Comment:
`rm -rf` command (or to be more precise - its arguments) is not compatible with Windows. We should consider creating a custom script with e.g.: `rimraf` package (that is already in our main node_modules) to cleanup directories without worrying if our unix-specific command will run somewhere else.

Reply:
Have we anyone who works on windows without a bash emulator like cmder? As this is a tool for stuff, should we care about it?

Reply:
Yes, me. Yes, we should care.

Reply:
It doesn't hold back the initial release. Let's do it at your earliest convenience @Siemienik